Tests/BasicsTests/Archiver/ZipArchiverTests.swift

```diff
@@ -256,18 +256,18 @@ class ArchiverTests: XCTestCase {
 }
 
 extension Archiver {
-    func extract(from: AbsolutePath, to: AbsolutePath) throws {
-        try temp_await {
+    func extract(from: AbsolutePath, to: AbsolutePath) async throws {
+        try await safe_async {
             self.extract(from: from, to: to, completion: $0)
         }
     }
```

Comment:
This function should be just removed, we already have an `async` version of it in `Sources/Basics/Archiver/Archiver.swift`

Reply:
done